src/lib/utils.js

```diff
@@ -73,3 +73,5 @@ export const loadCss = () => {
       document.documentElement.style.setProperty( storageKey, localStorage[storageKey] )
     ) )
 }
+
+export const timestamp = () => new Date( Date.now() ).toISOString().replace( /\.\d{3}\w$/, '' ).replace( 'T', ' ' )
```

Comment:
`new Date()` by default uses the current date and time

```suggestion
export const timestamp = () => new Date().toISOString().replace( /\.\d{3}\w$/, '' ).replace( 'T', ' ' )
```